trigger-dockerhub.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Patterned after https://github.com/bioperl/bioperl-live/blob/master/travis_scripts/trigger-dockerhub.sh
+
+set -e
+dockerapi="https://registry.hub.docker.com/u/dukegcb/bespin-web/trigger"
+
+
+if [[ -z "$DOCKERHUB_TOKEN" ]] ; then
+    echo "No API token for Docker Hub, add to repository settings."
+    exit 1
+fi
+
+echo "Triggering rebuild of Docker image dukegcb/bespin-web:latest"
+    curl -H "Content-Type: application/json" \
+         --data '{"docker_tag": "latest"}' \
```

Comment:
Looks good to me. I wonder if there's away to get the version number from the tag injected into the bespin-web build. Because right now that's hard-coded: https://github.com/Duke-GCB/bespin-webapp-docker/blob/438a6fa38a7c7076e7e6e25c622793721c39c526/bespin-web/Dockerfile#L27